contents.json

```diff
@@ -4976,6 +4976,11 @@
       "category": "fonts",
       "description": "An inkwell to use custom fonts on the fly.",
       "homepage": "https://github.com/ninjaprox/Inkwell"
+    }, {
+      "title": "InstantSearch iOS",
+      "category": "ui",
+      "description": "A library of widgets and helpers to build instant-search applications on iOS.",
```

Comment:
@spinach thanks for the contribution!
can you change "instant-search applications" to "instant-search feature"